launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml

```diff
@@ -230,7 +230,7 @@
     <include file="$(find-pkg-share object_merger)/launch/object_association_merger.launch.xml">
       <arg name="input/object0" value="$(var merger/input/objects)"/>
       <arg name="input/object1" value="clustering/camera_lidar_fusion/objects"/>
-      <arg name="output/object" value="camera_lidar_fusion/objects"/>
+      <arg name="output/object" value="camera_lidar_fusion/objects_without_tracker"/>
```

Comment:
How about `"$(var lidar_detection_model)_cluster_fusion/objects"`

Reply:
@miursh since it is result of roi, cluster and lidar_detection_model, so how about `"$(var lidar_detection_model)_roi_cluster_fusion/objects"`

Reply:
sounds good :+1:

Reply:
@miursh I renamed topic here https://github.com/autowarefoundation/autoware.universe/pull/3645/commits/bd9a1e815bb512ab0a205805b8c18197a0056780